docs/resources/elb_security_policy.md

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Dedicated Load Balance (Dedicated ELB)"
+---
+
+# flexibleengine_elb_security_policy
+
+Manages an ELB security policy resource within Flexibleengine.
+
+## Example Usage
+
+```hcl
+resource "flexibleengine_elb_security_policy" "test" {
+  name        = "security_policy_test"
+  description = "this is a security policy"
+  protocols   = ["TLSv1","TLSv1.1","TLSv1.2","TLSv1.3"]
+  ciphers     = ["ECDHE-RSA-AES256-GCM-SHA384","ECDHE-RSA-AES128-GCM-SHA256"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `protocols` - (Required, List) Specifies the TSL protocol list which the security policy select.
+  Value options: **TLSv1**, **TLSv1.1**, **TLSv1.2**, and **TLSv1.3**.
+
+* `ciphers` - (Required, List) Specifies the cipher suite list of the security policy.
+  The protocol and cipher suite must match. That is to say, there must be at least one cipher suite in
+  ciphers that matches the protocol. The following cipher suites are supported:
+  **ECDHE-RSA-AES256-GCM-SHA384**, **ECDHE-RSA-AES128-GCM-SHA256**, **ECDHE-ECDSA-AES256-GCM-SHA384**,
+  **ECDHE-ECDSA-AES128-GCM-SHA256**, **AES128-GCM-SHA256**, **AES256-GCM-SHA384**, **ECDHE-ECDSA-AES128-SHA256**,
+  **ECDHE-RSA-AES128-SHA256**, **AES128-SHA256**, **AES256-SHA256**, **ECDHE-ECDSA-AES256-SHA384**,
+  **ECDHE-RSA-AES256-SHA384**, **ECDHE-ECDSA-AES128-SHA**, **ECDHE-RSA-AES128-SHA**, **ECDHE-RSA-AES256-SHA**,
+  **ECDHE-ECDSA-AES256-SHA**, **AES128-SHA**, **AES256-SHA**, **CAMELLIA128-SHA**, **DES-CBC3-SHA**,
+  **CAMELLIA256-SHA**, **ECDHE-RSA-CHACHA20-POLY1305**, **ECDHE-ECDSA-CHACHA20-POLY1305**, **TLS_AES_128_GCM_SHA256**,
+  **TLS_AES_256_GCM_SHA384**, **TLS_CHACHA20_POLY1305_SHA256**, **TLS_AES_128_CCM_SHA256**,
+  **TLS_AES_128_CCM_8_SHA256**.
+
+* `name` - (Optional, String) Specifies the ELB security policy name.
+  The name contains only Chinese characters, letters, digits, underscores (_), and hyphens (-),
+  and cannot exceed 255 characters.
+
+* `description` - (Optional, String) Specifies the description of the ELB security policy.
+  The value can contain 0 to 255 characters.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project ID to which the Enterprise
+  router belongs.
+
+  Changing this parameter will create a new resource.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `listeners` - The listener which the security policy associated with.
+  The [ListenerRef](#SecurityPoliciesV3_ListenerRef) structure is documented below.
+
+<a name="SecurityPoliciesV3_ListenerRef"></a>
+The `ListenerRef` block supports:
```

Comment:
Usually, the field name, structure name and block name should keep consistent
```md
* `listeners` - The listener which the security policy associated with.
  The [listeners](#elb_listeners) structure is documented below.

<a name="elb_listeners"></a>
The `listeners` block supports:
```

Reply:
![image](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/assets/145660317/101a4eb7-75dd-4582-a785-784ca057b1c1)
done, i revise these three items to be consistent